### PR TITLE
Supplier uploads service definition scenarios

### PR DESCRIPTION
**Pull request by TheDoubleK**
These are new scenarios to test the new feature here: https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/755

### CODE REVIEW

File: features/supplier/view_and_edit_g_cloud_services.feature [hunk 1]
```diff
@@ -54,6 +54,30 @@ Scenario: Supplier user can edit the features and benefits of a service
...
+  Then I am on the 'Test cloud support service' page
+  And I see a success banner message containing 'You’ve edited your service. The changes are now live on the Digital Marketplace.'
+
+Scenario: Supplier user can not replace the service definition document of a service with a non-pdf file
+  When I click the top-level summary table Edit link for the section 'Documents'
+  Then I am on the 'Documents' page
+  And I choose file 'word.docx' for the field 'serviceDefinitionDocumentURL'
+  And I click 'Save and return to service'
+  Then I am on the 'Documents' page
+  And I see a validation message containing 'Your document exceeds the 5MB limit. Please reduce file size.'
```
Comment (katstevens) [inline features/supplier/view_and_edit_g_cloud_services.feature]: Is this the right validation message?

Comment (TheDoubleK) [reply features/supplier/view_and_edit_g_cloud_services.feature]: Sorry, no - I fixed this locally this morning but must have forgotten to push it.

**Comment by katstevens** [comment]
Looks good - the only other thing is whether we need `@skip-staging` tags for this feature?

**Comment by TheDoubleK** [comment]
Another good point... probably, yes 😞

**Comment by TheDoubleK** [comment]
Done.

**Pull request merged**